src/content/config.ts

```diff
@@ -29,4 +29,8 @@ const gallery = !import.meta.env.PUBLIC_CLOUDINARY_CLOUD_NAME
       }),
     });
 
-export const collections = { podcast, gallery };
+const team = defineCollection({
+  loader: file("src/components/about/team-members.json"),
+});
```

Comment:
To ensure we validate the team-members.json file at build time and with `astro check`, it's recommended to add a schema for your team loader here.
```
const team = defineCollection({
  loader: file("src/components/about/team-members.json"),
 schema: teamSchema // check blog schema for inspiration 
});
```